.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
Sure thing, not sure I'll get a chance to updating the PR tonight, but I'll get to it soon.